extensions/vscode/src/granite/ollama/ollamaLibrary.ts

```diff
@@ -12,8 +12,12 @@ export async function getRemoteModelInfo(
     return cache.get(modelId);
   }
   const start = Date.now();
-  const [modelName, tag] = modelId.split(":");
-  const url = `https://registry.ollama.ai/v2/library/${modelName}/manifests/${tag}`;
+  const [namespacedModelName, tag] = modelId.split(":");
+  const parts = namespacedModelName.split("/", 2);
+  const [namespace, modelName] =
+    parts.length === 2 ? parts : ["library", namespacedModelName];
```

Comment:
i guess this is okay, but to me the special case is `parts.length == 1` not `parts.length == 2`. Also, it's a little confusing that the code uses a variable called `namespacedModelName` only when the value in that variable has no namespace.  I think

```
const [namespace, modelName] =
   parts.length == 1 ? ["library", ...parts] : parts;
```
Would be clearer. Granted, I don't think we'll ever see a `parts.length > 2` so it doesn't really matter...